src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts

```diff
@@ -80,7 +80,7 @@ const GENERAL_VIRAL_UPLOAD_EXAMPLE_ROWS = [
   [
     EXAMPLE_SAMPLE_IDS[0], // sampleId
     "Private sample name", // privateId
-    "(if available) GenBank Accession", // publicId -- here as explainer
```

Comment:
@vincent-czi if we update the example row, current users won't have to update their upload template, right? I remember you saying the example rows are ignored during upload

Reply:
Yes. I just double-checked, that's right. All the data in an example row (that is, a row which starts with one of the EXAMPLE_SAMPLE_IDS) will be ignored during upload. So it's safe to change that description to whatever you want.